features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/NewVersion/NewVersion.jsx

```diff
@@ -182,6 +204,8 @@ class CreateNewVersion extends React.Component {
             helperText = 'An API with version "' + newVersion + '" already exists.';
         } else if (valid.version.hasSpecialChars) {
             helperText = 'API Version should not contain special characters';
+        } else if (valid.version.MaxLengthExceeds) {
+            helperText = 'API Version cannot be more than 30 characters';
```

Comment:
shall we change this to, API version exceed maximum length of 30 characters

Reply:
Done